data/categories.txt

```diff
@@ -3099,6 +3136,156 @@ sk:4Pomník|pamiatka|atrakcia|zaujímavosť
 sw:Makumbusho|kumbukumbu ya kihistoria|kivutio|utalii
 fa:یادبود تاریخی
 
+historic-memorial-cross
```

Comment:
BTW, need to investigate here.
I'm not sure about the behaviour if we have "@tourism" search tag in base class historic-memorial but don't have in derived class historic-memorial-*